tidb-cloud/branch-overview.md

```diff
@@ -41,6 +41,8 @@ Currently, TiDB Cloud Serverless branches are in beta and free of charge.
 
 - For each branch of a free cluster, 10 GiB storage is allowed. For each branch of a scalable cluster, 100 GiB storage is allowed. Once the storage is reached, the read and write operations on this branch will be throttled until you reduce the storage.
 
+- For each branch of a free cluster, you can only select `Include data up to` within 1 day. For each branch of a scalable cluster, you can select `Include data up to` within 14 days.
```

Comment:
```suggestion
- When [creating a branch](/tidb-cloud/branch-manage.md#create-a-branch) from a specific point in time:

    - For branches of a free cluster, you can select any time within the last 24 hours.
    - For each branch of a scalable cluster, you can select any time within the last 14 days.
```

Reply:
why the free cluster uses hours, but the scalable cluster uses days

Reply:
For a free cluster, if a user wants to create a branch at 10/18 18:00:
- "1 day": could be interpreted as either the previous calendar day (10/17 00:00-23:59) or the past 24 hours (10/17 18:00 - 10/18 18:00).
- "24 hours" clearly means 10/17 18:00 - 10/18 18:00

This change eliminates ambiguity for free clusters where precise hour-level timing matters. 

Regarding the "last 14 days" for scalable clusters, while it has similar ambiguity, it's less problematic because:
- users rarely need hour-level precision for 2-week periods
- the difference between interpretation is relatively small in the 14-day context

Reply:
get it